gravitee-management-api-rest/src/main/java/io/gravitee/management/rest/resource/ApiResource.java

```diff
@@ -134,6 +134,12 @@ public Response doLifecycleAction(@QueryParam("action") LifecycleActionParam act
     @Produces(MediaType.APPLICATION_JSON)
     @ApiPermissionsRequired(ApiPermission.MANAGE_API)
     public ApiEntity update(@Valid @NotNull final UpdateApiEntity api) {
+        final ApiEntity currentApi = this.get();
+        // Force context-path if user is not the primary_owner or an administrator
+        if (currentApi.getPermission() != MembershipType.PRIMARY_OWNER) {
```

Comment:
If you have MANAGE_API permission but you're not the primary_owner, you reset the context-path.